### ISSUE TITLE
See screenshot

### ISSUE THREAD

**Issue by xIFAILx** [PocketMobsters/PokeMobBot#480]
![image](https://cloud.githubusercontent.com/assets/5966646/17348056/2b9f8500-5914-11e6-8a52-4761eed955d3.png)

**Comment by prpltntl** [comment]
Getting the same thing

**Comment by devoknows** [comment]
Same

**Comment by devoknows** [comment]
Try logging into another account on PokeMobBot, then log back into your original account. I was able to log in w/o the error doing that.

**Comment by xIFAILx** [comment]
I just waited for a bit and ran it again, also seemed to work, next time i'll try the different account thing tho

**Comment by devoknows** [comment]
@xIFAILx my method prob had nothing to do with the fix then tbh. Prob just needed to wait to run again.

**Comment by R0DaV-D0rIaN** [comment]
I tried both methods, it is not working

**Comment by R0DaV-D0rIaN** [comment]
It is a human error. When you look up the coordinates, sometimes it tells you the longitude incorrectly. Try going to Google, and looking up the exact coordinates that you put in. If it is not the location that you intended, then make the longitude negative.

**Comment by prpltntl** [comment]
I don't think that fits with most of our reported issue. Mine was working for hours then stopped, so the coords in the config definitely weren't a factor.  And the people saying they waited and then retried and it stopped would have probably mentioned if they changed the coords too. I did the same, quit, walked away for about an hour, then when I came back its been working, definitely didn't change anything, since it had been working with my settings for hours I just left them the same.
